components/composer/src/Composer.svelte

```diff
@@ -619,21 +595,21 @@
             {#if minimized}
               <button class="composer-btn" on:click={handleMinimize}>
                 <ExpandIcon
-                  style="fill: var(--icons); width: 10px; height: 10px; transform: translateY(1px)"
+                  style="fill: var(--composer-icons-color, #666774); width: 10px; height: 10px; transform: translateY(1px)"
```

Comment:
Can we move these styles out of the markup, they're getting a bit long? Same with all of the icon styles in composer.